kie-dmn-api/src/main/java/org/kie/dmn/api/core/ast/DMNExpressionEvaluator.java

```diff
@@ -32,27 +32,10 @@
      *
      * @return the result of the evaluation of the expression
      */
-    EvaluatorResult evaluate(InternalDMNRuntimeEventManager eventManager, DMNResultImpl result);
+    EvaluatorResult evaluate(InternalDMNRuntimeEventManager eventManager, InternalDMNResult result);
 
     enum ResultType {
         SUCCESS, FAILURE;
     }
-
-    class EvaluatorResult {
```

Comment:
`EvaluatorResult` is now a non-child interface, which later kie-dmn-core provide a concrete implementation `EvaluatorResultImpl` for.